### PR TITLE
Add Chainlink

### LINKED ISSUE #3812: chainlink
**Name:** Chainlink
**Website:** https://chain.link/
**Alexa rank:** #35,390
  

**Official resources for icon and color:** https://chain.link/assets/images/icon-logo-white-ef878c98.svg white

### PR DESCRIPTION
**Pull request by adamalston**
chainlink

Closes #3812 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Color from source css

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1285,6 +1285,11 @@
             "hex": "1EABE2",
             "source": "https://cevo.com/"
         },
+        {
+            "title": "Chainlink",
+            "hex": "2A5ADA",
+            "source": "https://chain.link"
```
Comment (adamrusted) [inline _data/simple-icons.json:1291]: Icon looks spot on to me - is it worth linking directly to the [brand assets](https://chain.link/brand-assets) page? That contains more info about the colours etc, so would be more helpful to users I feel.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -1285,6 +1285,11 @@
             "hex": "1EABE2",
             "source": "https://cevo.com/"
         },
+        {
+            "title": "Chainlink",
+            "hex": "2A5ADA",
```
Comment (adamrusted) [inline _data/simple-icons.json:1290]: On the [brand assets](https://chain.link/brand-assets) page, the primary brand colour is described as `#375BD2`. Would you be able to update it here too?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for your work on this @adamalston - just a few quick changes listed below.

File: _data/simple-icons.json (hunk 1 above)
Comment (adamalston) [reply _data/simple-icons.json]: Updated

**Review by adamrusted — APPROVED** [review]
Looks good to me, thanks @adamalston 🎉

File: _data/simple-icons.json (hunk 2 above)
Comment (adamalston) [reply _data/simple-icons.json]: Updated

**Pull request merged**